client/my-sites/email/email-management/gsuite-user-item/index.jsx

```diff
@@ -20,40 +20,67 @@ import './style.scss';
 function GSuiteUserItem( props ) {
 	const translate = useTranslate();
 	const [ dialogVisible, setDialogVisible ] = useState( false );
+
 	const onFixClickHandler = ( e ) => {
 		e.preventDefault();
 		setDialogVisible( true );
 	};
+
 	const onCloseClickHandler = () => {
 		setDialogVisible( false );
 	};
 
+	const getMailboxLink = () => {
+		const { email } = props.user;
+
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://mail.google.com/`;
```

Comment:
The url should be:

https://accounts.google.com/AccountChooser?service=mail&continue=https://mail.google.com/mail/.